### ISSUE TITLE
Focussed link color contrast is bad in dark mode

### ISSUE THREAD

**Issue by jsoref** [argoproj/argo-cd#29288]
Checklist:

- [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
- [x] I've included steps to reproduce the bug.
- [x] I've pasted the output of `argocd version`.

**Describe the bug**

Link color is dark blue text on dark gray background

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/applications/argocd/appset-progressive-prod-us-west?view=tree&resource=
2. Tab to focus the `from` link (e.g. `HEAD
(8088f4c)`)

**Expected behavior**

Links should be readable when focussed

**Screenshots**

<img width="1049" height="508" alt="Image" src="https://github.com/user-attachments/assets/0cc84f34-2264-49e9-a19a-42de3314ce5c" />

**Version**

```shell
{
    "Version": "v3.6.0+b5fc12e",
    "BuildDate": "2026-08-13T10:22:54Z",
    "GitCommit": "b5fc12e0155c6a58c0924afc0ec68478cf25da11",
    "GitTreeState": "clean",
    "GoVersion": "go1.26.5",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KustomizeVersion": "v5.8.1 2026-02-09T16:15:27Z",
    "HelmVersion": "v4.2.3+g43e8b7f",
    "KubectlVersion": "v0.36.1",
    "JsonnetVersion": "v0.22.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```
